### DISCUSSION TITLE
Need to zoomin/zoomout the web page using webdriverio

### DISCUSSION THREAD

**Discussion by ayubansal01** [webdriverio/webdriverio#8391] [Q&A]
Hi,

Could some one please help to figure out how to zoom in and zoom out the webpage or browser using webdriverio . I have tried using browser.keys(['Control', '-']) but it's not working.

**Comment by erwinheitzman** [comment]
You can use the execute command to inject the following: document.body.style.zoom = '30%'

**Reply by ayubansal01** [reply]
This is working when browser is initialized or user login the site but after page is loaded then page sets to original size.

**Reply by erwinheitzman** [reply]
You should execute it after the page load

**Reply by ayubansal01** [reply]
Already tried , not working :(

**Reply by kopax-polyconseil** [reply]
Any update on this ? I am be interested to allow 200% zoom test

**Reply by erwinheitzman** [reply]
The following argument is for chrome: --zoom=0.8

There's probably more ways to do this as well, I just did a quick Google search to find this example

**Reply by erwinheitzman** [reply]
Also have you tried the original suggestion I gave? It worked fine for me

**Reply by MuthuITQA** [reply]
> You can use the execute command to inject the following: document.body.style.zoom = '30%'

This is working fine but after reducing the size of webpage unable to identify webelements, can any one help me on this

**Comment by MuthuITQA** [comment]
I tried still it is  not working 

 await browser.url("https://the-internet.herokuapp.com/key_presses");
       browser.keys(["Control", "-", "Null"]);

**Comment by erwinheitzman** [comment]
Apperently the zoom property is not cross browser compatible, you can use this instead:
```js
document.body.style.transform = 'scale(2)';
```

**Comment by MuthuITQA** [comment]
Exdcute command (document.body.style.zoom = '30%') is wokring but after the webpage is zoom in 30% is completed unable to find the webelements
